fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/IndexWriterCommitCheckAsync.java

```diff
@@ -82,6 +83,7 @@ protected void doMerge(final IndexWriter writer, final MergePolicy.OneMerge merg
             }
         });
         indexWriterConfig.setCodec(new LuceneOptimizedCodec(Lucene50StoredFieldsFormat.Mode.BEST_COMPRESSION));
+        indexWriterConfig.setInfoStream(new PrintStream(new LoggerOutStream(LOGGER), false));
```

Comment:
Why not just have a `LoggerInfoStream` and use the other overload `IndexWriterConfig.setInfoStream(InfoStream infoStream)`?

Reply:
Yeah that's a good point. Re-implemented it